src/NLog/LayoutRenderers/Wrappers/WrapperLayoutRendererBuilderBase.cs

```diff
@@ -1,107 +0,0 @@
-// 
-// Copyright (c) 2004-2020 Jaroslaw Kowalski <jaak@jkowalski.net>, Kim Christensen, Julian Verdurmen
-// 
-// All rights reserved.
-// 
-// Redistribution and use in source and binary forms, with or without 
-// modification, are permitted provided that the following conditions 
-// are met:
-// 
-// * Redistributions of source code must retain the above copyright notice, 
-//   this list of conditions and the following disclaimer. 
-// 
-// * Redistributions in binary form must reproduce the above copyright notice,
-//   this list of conditions and the following disclaimer in the documentation
-//   and/or other materials provided with the distribution. 
-// 
-// * Neither the name of Jaroslaw Kowalski nor the names of its 
-//   contributors may be used to endorse or promote products derived from this
-//   software without specific prior written permission. 
-// 
-// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE 
-// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE 
-// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE 
-// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR 
-// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
-// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS 
-// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN 
-// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) 
-// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF 
-// THE POSSIBILITY OF SUCH DAMAGE.
-// 
-
-using System;
-using System.Text;
-
-namespace NLog.LayoutRenderers.Wrappers
-{
-    /// <summary>
-    /// Base class for <see cref="LayoutRenderer"/>s which wrapping other <see cref="LayoutRenderer"/>s. 
-    /// 
-    /// This expects the transformation to work on a <see cref="StringBuilder"/>
-    /// </summary>
-    public abstract class WrapperLayoutRendererBuilderBase : WrapperLayoutRendererBase
```

Comment:
As this one was never obsolete, I propose to add the obsolete attribute and it's OK too remove it in NLog 6.

Reply:
No one is using it. So see no reason to not cleanup as much as possible in NLog 5

Reply:
How do we know that? It's public. We clean as much as possible by not using it?

Reply:
Thought the whole idea with Major versions was the ability to make breaking changes. And not being required to make obsolete first. Especially when extremely rare features.

Reply:
Yes breaking changes are allowed, but Obsolete make this decent.

I don't see the issue in let this class live for now. I won't influence other code. It's only ugly...

Reply:
Please image your are a enduser, uses only public classes, not warnings with obsoletes and suddenly the type is gone. We can't even show a nice message then.

Reply:
I'm guessing that a full chapter has be written where explaining all breaking changes, and new default values. This should be read careful when upgrading from old version to NLog 5. But I will add the class back and mark it as obsolete.